geomesa-gs-catalog-listener/pom.xml

```diff
@@ -36,6 +36,14 @@
             <artifactId>accumulo-core</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.geoserver</groupId>
+            <artifactId>gs-main</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.geoserver</groupId>
+            <artifactId>gs-platform</artifactId>
+        </dependency>
```

Comment:
Can this file be reverted?

Reply:
Maybe it is tied to something I had changed in the root pom.xml.  One of the two places can be cleaned up.